client/src/api/README.md

```diff
@@ -0,0 +1,15 @@
+# API
+
+This folder contains API functions used to power the DAShboard.
+
+## [V2 Bike Specific API](./v2)
+
+This folder contains API functions that are ***ONLY* for the V2 bike**.
+
+## [V3 Bike Specific API](./v3)
+
+This folder contains API functions that are ***ONLY* for the V3 bike**.
+
+## [Common API](./common)
+
+This folder contains API functions that are ***SHARED* between V2 and V3 bikes**.
```

Comment:
It's good being explicit but I reckon it wouldn't hurt to write "all" rather than "V2 and V3" just so it doesn't need to be updated in the future. (applies everywhere else too)

Reply:
I wasn't sure but I think now that you said it. It is a better approach.